### ISSUE TITLE
Installing Liquid Soap on Jessie with libaacplus-ocaml-dynlink failed

### ISSUE THREAD

**Issue by sdt105** [savonet/liquidsoap#233]
Hi,

i've installed Liquid Soap on Debian Jessie via apt-get (V 1.1.1). After that i added deb-multimedia.org in my sources.list file and installed libaacplus-ocaml-dynlink.

But when i'm using aacplus in liquid soap now, the aacplus encoder cannot be found ("No encoder found for that format").

What's wrong with that and is it possible to install Liquid Soap on Debian Jessie without the need for recompile?

Thanks for an answer!

Edit: version of libaacplus-ocaml-dynlink is 0.2.1

**Comment by shortyxs** [comment]
Hi there,
I'm recentyl Meeting this Problem and can't finde any solution. It seems anybody who met this Problem before, gave up working on it. Is there any way to get %aacplus or %fdk-aac working on jessie?`
I checked the Folders, that liquidsoap log mentions and it seems all required files are presen.

Thanks and best regards
Malte

**Comment by toots** [comment]
Hi @shortyxs I would recommend installing liquidsoap with `opam` as describe here: http://liquidsoap.fm/download.html Then, if your Debian distribution is recent enough (`>= jessie`), you should be able to compile liquidsoap with fdkaac enabled using the `opam` `fdkaac` package and the Debian `libfdk-aac-dev` package.

**Comment by shortyxs** [comment]
Hi @toots ,
I am system administrator not a software developer, so I have no clue how to compile an application. Why can't I just apt-get install as any other application?
As mentioned already, I have the exact same problem as the threat srarter. I also did the exact same steps on the exact same versions and I did that before I found there is an open issue here on Github.

I tried the opam guide, but any opam command fails with an error.
